cli/internal/runsummary/spaces.go

```diff
@@ -19,11 +19,11 @@ type spacesRunPayload struct {
 	Command        string `json:"command,omitempty"`        // the thing that kicked off the turbo run
 	RepositoryPath string `json:"repositoryPath,omitempty"` // where the command was invoked from
 	Context        string `json:"context,omitempty"`        // the host on which this Run was executed (e.g. Github Action, Vercel, etc)
+	GitBranch      string `json:"gitBranch"`
+	GitSha         string `json:"gitSha"`
```

Comment:
The prefix here says that we should maybe consider pushing this into an object with properties. That could make it forward-compatible:

```js
{
  scm: {
    type: "git",
    branch: "foo",
    sha: "C0FFEE"
  }
}
```

We do it this way in run summary, so it's a _bit_ weird to depart from that.

Reply:
This is just where the API is right now. I agree we could change it but there are other departures from current run summary structure also, so I don't want to tug on this thread right now